File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task25_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class Main {
    public static void main(String[] args) {
        String[] lines = {"a", "b", "c"};
        for (String line : numberLines(lines)) {
            System.out.println(line);
        }
    }

    public static String[] numberLines(String[] lines) {
        for (int i = 0; i < lines.length; i++) {
            lines[i] = (i + 1) + ": " + lines[i];
        }
        return lines;
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task25_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <vector>
#include <string>

int main() {
    std::vector<std::string> lines = {"a", "b", "c"};
    for (int i = 0; i < lines.size(); i++) {
        std::cout << (i + 1) << ": " << lines[i] << std::endl;
    }
    return 0;
}